LFGBulletinBoard/LibGPIToolBox.lua

```diff
@@ -51,6 +51,16 @@ Tool.RoleIcon = {
 Tool.Classes=CLASS_SORT_ORDER
 Tool.ClassName=LOCALIZED_CLASS_NAMES_MALE
 Tool.ClassColor=RAID_CLASS_COLORS
+-- support for CUSTOM_CLASS_COLORS
+if CUSTOM_CLASS_COLORS then
+	for k, v in pairs(CUSTOM_CLASS_COLORS) do
+		---@cast v ColorMixin
+		if not v.colorStr then
+			v.colorStr = v:GenerateHexColor();
```

Comment:
ColorMixins do not have the `colorStr` member by default. It's something that blizz manually adds onto the `RAID_CLASS_COLORS` entries. 

Custom color addons *should* take care of adding this field. Its worth verifying here however since this addon relies heavily `colorStr` for coloring text.